src/main/java/com/facebook/openwifirrm/ucentral/informationelement/LocalPowerConstraint.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.informationelement;
+
+import java.util.Objects;
+
+import com.google.gson.JsonObject;
+
+/**
+ * This information element (IE) appears in wifiscan entries. It is called
+ * "Local Power Constraint" in these entries, and just "Power Constraint" in
+ * the 802.11 specification. Refer to the specification for more details.
+ * Language in javadocs is taken from the specification.
+ */
+public class LocalPowerConstraint {
+
+	/** Defined in 802.11 */
+	public static final int TYPE = 32;
+
+	/**
+	 * Units are dB.
+	 * <p>
```

Comment:
Why there is `<p>`?

Reply:
It is a javadoc formatting thing. It adds a new paragraph.